cloudflare/Cargo.toml

```diff
@@ -23,10 +23,4 @@ async-graphql-value = "7.0.1"
 serde_json = "1.0.113"
 serde_qs = "0.12.0"
 console_error_panic_hook = "0.1.7"
-protox = "0.5.1"
-
-[profile.release]
-lto = true
-strip = true
-codegen-units = 1
-opt-level = 'z'
```

Comment:
We need to add these configurations somewhere. This needs to be used during our release builds.